src/main/java/com/google/gerrit/extensions/api/changes/ChangeApi.java

```diff
@@ -187,6 +188,7 @@ SubmittedTogetherInfo submittedTogether(
   Map<String, List<CommentInfo>> drafts() throws RestApiException;
 
   ChangeInfo check() throws RestApiException;
+  IncludedInInfo in() throws RestApiException;
```

Comment:
Please add this method with the same name (`includedIn`) and at the same place (below `topic(...)`) as it got added in the Gerrit extensions API:
https://github.com/gerrit-review/gerrit/blob/master/gerrit-extension-api/src/main/java/com/google/gerrit/extensions/api/changes/ChangeApi.java#L159